pkg/interactive/interactive_client.go

```diff
@@ -347,6 +347,10 @@ func (c *InteractiveClient) runInteractivePrompt(ctx context.Context) (ret utils
 		}),
 		// ignore suppressed ASCII codes
 		prompt.OptionAddASCIICodeBind(c.suppressOnInput()...),
+		prompt.OptionAddKeyBind(prompt.KeyBind{
```

Comment:
Let's not add this. This may interfere with the other handler for `Ctrl+C` which is responsible for breaking out of a multiline session. Moreover I don't think we want to toggle auto complete based on Ctrl+C. `Ctrl+C` is more of a `cancel current action` than `disable/enable this feature`